### PR TITLE
[stable/rabbitmq-ha] Added support for extraLables in the ingress.yaml template

### PR DESCRIPTION
**Pull request by mmojica53**
#### What this PR does / why we need it:
This adds the ability of specifying labels in ingress templates
#### Which issue this PR fixes
It doesn't not fix any known issue

#### Special notes for your reviewer:
It's a very simple change in the ingress template
#### Checklist
- [x ] Variables are documented in the README.md
- [x ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by mmojica53** [comment]
I updated my commit to include the version bump, sorry about that

**Review by steven-sheehy — APPROVED** [review]
/assign
/ok-to-test

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Need to bump the chart version as another PR was merged. Also, please sign your commits for DCO.

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
-appVersion: 3.8.5
-version: 1.46.3
+appVersion: 3.8.6
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: Please don't bump the appVersion as it's still at rabbitmq 3.8.5

**Review by steven-sheehy — APPROVED** [review]
/lgtm
Thanks!

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (mmojica53) [reply stable/rabbitmq-ha/Chart.yaml]: Sorry about that, this is my first time doing this, I updated the commit

**Pull request merged**